src/components/ConfirmContent.js

```diff
@@ -36,6 +36,9 @@ const propTypes = {
     /** Whether we should show the cancel button */
     shouldShowCancelButton: PropTypes.bool,
 
+    /** Whether the cancel button has hover effect */
+    cancelWithHoverEffect: PropTypes.bool,
```

Comment:
https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#boolean-variables-and-props

>Boolean props or variables must be prefixed with should or is to make it clear that they are Boolean. Use should when we are enabling or disabling some features and is in most other cases.

We should change this with the `should` prefix.

Reply:
Updated to `shouldCancelBtnHaveHoverEffect`